### DISCUSSION TITLE
Showcase: AXIVO website powered by Hextra

### DISCUSSION THREAD

**Discussion by fmunteanu** [imfing/hextra#397] [Show and tell]
Please let me know if you have any suggestions or improvements. @imfing please feel free to add the website to your showcase, if you find the design original.

Main Project: https://axivo.com/k3s-cluster/, the goal is add several other projects while using the same Hextra template
Root: https://axivo.com

> [!TIP]
> Credit is provided into [README](https://github.com/axivo/website/blob/main/README.md)

Related discussion: #396, trying to use a base template for both root and main project
Related PR's: #389 #390

**Comment by imfing** [comment]
Awesome work! 🎉 

> please feel free to add the website to your showcase

Could you send a quick PR to add to the showcase? Thanks!

**Comment by fmunteanu** [comment]
@imfing I created https://github.com/imfing/hextra/pull/400.

**Comment by fmunteanu** [comment]
@imfing I'm curious why the showcase card is cropping the top of the image?

**Reply by imfing** [reply]
the image aspect ratio is set to 16/9

**Reply by fmunteanu** [reply]
@imfing I created a new PR. https://github.com/imfing/hextra/pull/402
